### PR TITLE
delete garbage

### PR DESCRIPTION
**Pull request by Mallusca**

### CODE REVIEW

File: SchoolJournalCore.BLL/JournalGridColumnsService.cs [hunk 1]
```diff
@@ -26,10 +26,7 @@ public class JournalGridColumnsService : IJournalGridColumnsService
             _columnsRepository = columnsRepository;
             _columnMarksRepository = columnMarksRepository;
             _columnTypeRepository = columnTypeRepository;
-        }
-        
-        //public deleteRow()
-        //delete student+delete marks
+        }
```
Comment (AJIex1s) [inline SchoolJournalCore.BLL/JournalGridColumnsService.cs:30]: add new line

File: SchoolJournalCore.BLL/JournalGridColumnsService.cs [hunk 2]
```diff
@@ -26,10 +26,7 @@ public class JournalGridColumnsService : IJournalGridColumnsService
             _columnsRepository = columnsRepository;
```
Comment (AJIex1s) [inline SchoolJournalCore.BLL/JournalGridColumnsService.cs:26]: add new line

File: SchoolJournalCore.BLL/JournalGridColumnsService.cs (hunk 1 above)
Comment (Mallusca) [reply SchoolJournalCore.BLL/JournalGridColumnsService.cs:30]: fixed

File: SchoolJournalCore.BLL/JournalGridColumnsService.cs (hunk 2 above)
Comment (Mallusca) [reply SchoolJournalCore.BLL/JournalGridColumnsService.cs:26]: fixed

File: SchoolJournalCore.BLL/JournalGridColumnsService.cs [hunk 3]
```diff
@@ -26,10 +26,7 @@ public class JournalGridColumnsService : IJournalGridColumnsService
...
             _columnMarksRepository = columnMarksRepository;
             _columnTypeRepository = columnTypeRepository;
-        }
-        
-        //public deleteRow()
-        //delete student+delete marks
```
Comment (AJIex1s) [inline SchoolJournalCore.BLL/JournalGridColumnsService.cs:32]: fix it

**Pull request merged**